lib/dish.rb

```diff
@@ -0,0 +1,7 @@
+class Dish
+    attr_reader :name, :price
+    def initialize(name, price)
+        @price = price
+        @name = name
+    end
+end
```

Comment:
This class could be useful later down the line, but for now it could probably be replaced with a simple hash: `{ name: "Pizza", price: "7.00" }`